utam-core/src/main/java/utam/core/selenium/appium/MobileDriverAdapter.java

```diff
@@ -75,10 +78,18 @@ final AppiumDriver switchToWebView(String title) {
     Set<String> contextHandles = appiumDriver.getContextHandles();
     for (String contextHandle : contextHandles) {
       if (!contextHandle.equals(NATIVE_CONTEXT_HANDLE)) {
-        AppiumDriver newDriver = (AppiumDriver) appiumDriver.context(contextHandle);
-        String newTitle = newDriver.getTitle();
-        if (!newTitle.isEmpty() && newTitle.equalsIgnoreCase(title)) {
-          return newDriver;
+        AppiumDriver newDriver;
+        try {
+          newDriver = (AppiumDriver) appiumDriver.context(contextHandle);
+        } catch (WebDriverException e) {
+          warning("Context switch to webview '" + contextHandle + "' failed. Error : " + e.getMessage());
```

Comment:
nit: pls use `String.format("...", contextHandle, e.getMessage())` for consistency with other code and remove space before semicolon

Reply:
@lizaiv77 have updated.